content/en/developers/metrics/distributions.md

```diff
@@ -11,19 +11,36 @@ further_reading:
   text: "Official and Community-contributed API and DogStatsD client libraries"
 ---
 
-<div class="alert alert-warning">
-This feature is in beta. <a href="/help">Contact Datadog support</a> to enable distribution metrics for your account.
-</div>
-
 ## Overview
 
-Distributions measure the statistical distribution of a set of values across one or more hosts. In this way, Distributions are like a "global" version of [Histograms][1].
+Distributions are a new metric type in Agent 6 that aggregate the values that are sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure. This can be thought of as a global version of our existing [Histogram metric type][1], which measures the statistical distribution of values on a single host.
```

Comment:
```suggestion
Distributions are a new metric type in Agent v6 that aggregate the values that are sent from multiple hosts during a flush interval to measure statistical distributions across your entire infrastructure. This can be thought of as a global version of our existing [Histogram metric type][1], which measures the statistical distribution of values on a single host.
```
Does the customer need a specific Agent version or higher (example v6.12+)?